setup.cfg

```diff
@@ -1,5 +1,5 @@
 [tool:pytest]
-DJANGO_SETTINGS_MODULE = tests.test_config
+DJANGO_SETTINGS_MODULE = justspaces.settings
```

Comment:
Is this change just because the test settings weren't changing anything? Do the tests not use a separate DB?

Reply:
Yep, they were the same and it seemed like an unnecessary hassle to have to change settings in two places.